src/main/java/org/folio/utils/UserPermissionsUtil.java

```diff
@@ -63,12 +67,31 @@ public static void verifyUserHasManagePermission(List<String> newAcqUnitIds, Lis
     }
   }
 
+ public static void  verifyUserHasInvoicePayPermission  (Invoice.Status newInvoiceStatus, Invoice.Status statusFromStorage,   Map<String, String> okapiHeaders ){
+
+    if (isInvoiceStatusUpdated(newInvoiceStatus, statusFromStorage)){
+      if(!(newInvoiceStatus.toString().equals("Approved")) &&!(newInvoiceStatus.toString().equals("Paid"))&& isUserDoesNotHaveDesiredPermission(PAY, okapiHeaders)){
+      throw new HttpException(HttpStatus.HTTP_FORBIDDEN.toInt(),  CANNOT_PAY_INVOICE_WITHOUT_APPROVAL);
+    }
+ }
+ }
+
+  public static void verifyUserHasInvoiceApprovePermission (Invoice.Status newInvoiceStatus, Invoice.Status statusFromStorage, Map<String, String> okapiHeaders ){
+
+    if (isInvoiceStatusUpdated(newInvoiceStatus, statusFromStorage ) && isUserDoesNotHaveDesiredPermission(APPROVE, okapiHeaders)){
+
```

Comment:
lets use the same code style, these empty new lines not needed here

Reply:
will remove the new lines